doc/man3/EVP_DigestSignInit.pod

```diff
@@ -166,6 +166,10 @@ external circumstances (see L<RAND(7)>), the operation will fail.
 The call to EVP_DigestSignFinal() internally finalizes a copy of the digest
 context. This means that calls to EVP_DigestSignUpdate() and
 EVP_DigestSignFinal() can be called later to digest and sign additional data.
+Note that not all providers allow for continuation, in case the selected
+provider does not allow to duplicate contexts EVP_DigestSignFinal() will
+finalize the digest context and attempting to process additional data via
+EVP_DigestSignUpdate() will result in an error.
```

Comment:
That sentence should probably be split up. I think it's probably also better to say "support" rather than "allow for".

We should probably also point to EVP_MD_CTX_set_flags() where they can change that behaviour with the EVP_MD_CTX_FLAG_FINALISE flag.

Reply:
Do you have any suggestion for how to word about EVP_MD_CTX_FLAG_FINALISE ?

Reply:
@kroeckx Please see if the language in the fixup comment I pushed addresses this request.